main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,177 @@
+`sudo usermod -aG docker $USER`
 
-After your solo wallet is connected, then you're ready to deploy the contract and API. 
+![029](./assets/029.png)
 
-## Deploy the Contract and API
+Now go ahead and reboot your image.
 
-In our second terminal, deploy the contract to the simulated blockchain
-and the API to the solo client.
+![030](./assets/030.png)
 
-```sh secondary style2
-# Terminal 2
-cd demo # if not already there
-agoric deploy ./contract/deploy.js 
-agoric deploy ./api/deploy.js
-```
+Once your image has rebooted and you've logged back on, test that Docker works by running the `hello-world` sample.
+`docker run hello-world`
 
-We'll cover [deploying smart contracts](./deploying.md)
-in detail later.
+![031](./assets/031.png)
 
-## Start the Dapp User Interface
 
-The web user interface communicates with the API in the solo client as well as the wallet.
+# Installing the Sample Dapp
+Now you'll use yarn to pull down the sample dapp. The sample dapp will be placed in a subfolder named `demo`.
+`yarn create @agoric/dapp --dapp-base https://github.com/agoric-labs/ --dapp-template dapp-game-places demo`
 
-```sh secondary style3
-# Terminal 3
-cd demo # if not already there
-cd ui && yarn start
-```
+![024](./assets/024.png)
+
+
+# Install Dapp Dependencies
+Now navigate to the `demo` directory and run the `yarn install` command to install any solution dependencies.
+`cd demo`
+`yarn install`
+
+![033](./assets/033.png)
+
+
+# Starting the Network
+Now go ahead and start the network using the `yarn start` command.
+`yarn start:docker`
+
+![034](./assets/034.png)
+
+Once the network has started you can check the logs. Once you see messages showing blocks with a status of `commit` you can rest assured the network is running properly.
+`yarn docker:logs`
+
+![035](./assets/035.png)
+
+
+# Starting the Dapp Smart Contract
+Exit the log window and start the smart contract by running the `yarn start` command.
+`yarn start:contract`
+
+![036](./assets/036.png)
+
+
+# Installing Keplr Wallet
+Next, you'll install the Keplr wallet plug-in. Open up your browser and navigate to [https://www.keplr.app/download](https://www.keplr.app/download). Select the version appropriate to your browser. 
+
+![037](./assets/037.png)
+
+Once the plug-in has been installed, open Keplr and select the option to "Import an existing wallet". Then choose the option to "Use recovery phrase or private key".
+
+![038](./assets/038.png)
 
-Leave this running in its own terminal window and visit [http://localhost:3000](http://localhost:3000) in a web browser.
+![040](./assets/040.png)
 
-## Connect the Dapp to the Agoric Wallet
+To import your wallet, you'll need to copy your mnemonic phrase into Keplr. You can find this series of 24 words back on your terminal window. Copy from this window into your Keplr wallet, then hit the "Import" button. Note that your phrase will NOT be the same as the one shown in this guide, all phrases are unique!
```

Comment:
The phrase might well be the same as what's in these docs. It only changes when there's a new agoric-3-proposals image.

And again, due to the importance of security to our brand, it's important to be explicit that the handling of the mnemonic phrase in this tutorial doesn't follow best practices. The way I wrote it in #881 was:

https://github.com/Agoric/documentation/blob/312d426b33d1f5be23d6d40fc980b2d801728390/main/guides/getting-started/README.md?plain=1#L197-L203

Reply:
^^ Noted